### PR TITLE
modified eligibility condition

### PR DESCRIPTION
**Pull request by xenosnikos**

### CODE REVIEW

File: src/widgets/PaymentPlans/PaymentPlans.tsx [hunk 1]
```diff
@@ -36,24 +36,26 @@ export class PaymentPlans extends Widget<PaymentPlanSettings> {
...
+​
     const { purchaseAmount, plans } = this.config
     const installmentsCountsToQuery = []
-
+​
     // For each plan to be queried, check whether it's worth querying (i.e. is the purchase amount
     // is within the plan's boundaries) and act accordingly
     for (const plan of plans) {
+
+      if(plan.minAmount != 0){
```
Comment (syjust) [inline src/widgets/PaymentPlans/PaymentPlans.tsx:58]: indent and add spaces between `if` `(` and `{` here please

File: src/widgets/PaymentPlans/PaymentPlans.tsx [hunk 2]
```diff
@@ -68,13 +70,20 @@ export class PaymentPlans extends Widget<PaymentPlanSettings> {
             },
           }),
         )
-      } else {
+        }else{
```
Comment (syjust) [inline src/widgets/PaymentPlans/PaymentPlans.tsx:73]: indent and add spaces between `if` `(` and `{` here please

File: src/widgets/PaymentPlans/PaymentPlans.tsx [hunk 3]
```diff
@@ -68,13 +70,20 @@ export class PaymentPlans extends Widget<PaymentPlanSettings> {
...
+           installmentsCountsToQuery.push(plan.installmentsCount)
+           results.push(installmentsCountsToQuery.length - 1)
+         }
+
+      }
+      else {
```
Comment (syjust) [inline src/widgets/PaymentPlans/PaymentPlans.tsx:79]: please, don't jump line for `else` statement

File: src/widgets/PaymentPlans/PaymentPlans.tsx [hunk 4]
```diff
@@ -125,4 +134,4 @@ export class PaymentPlans extends Widget<PaymentPlanSettings> {
       />
     )
   }
-}
+}
```
Comment (syjust) [inline src/widgets/PaymentPlans/PaymentPlans.tsx:137]: leave `\n` at EOF please (modify only part of code you have to)

File: src/widgets/PaymentPlans/PaymentPlans.tsx [hunk 5]
```diff
@@ -4,22 +4,22 @@ import { PaymentPlanSettings } from './types'
 import { DefaultWidgetConfig, SettingsLiteral } from '@/widgets/config'
 import { PaymentPlansRenderer } from '@/widgets/PaymentPlans/PaymentPlansRenderer'
 import { IEligibility } from '@alma/client/dist/types/entities/eligibility'
-
+​
```
Comment (syjust) [inline src/widgets/PaymentPlans/PaymentPlans.tsx:7]: remove line please (modify only part of code you have to)

File: src/widgets/PaymentPlans/PaymentPlans.tsx [hunk 6]
```diff
@@ -4,22 +4,22 @@ import { PaymentPlanSettings } from './types'
...
 type PaymentPlanDefaultConfig = DefaultWidgetConfig<PaymentPlanSettings>
-
+​
```
Comment (syjust) [inline src/widgets/PaymentPlans/PaymentPlans.tsx:9]: remove line please (modify only part of code you have to)

File: src/widgets/PaymentPlans/PaymentPlans.tsx [hunk 7]
```diff
@@ -4,22 +4,22 @@ import { PaymentPlanSettings } from './types'
...
 export class PaymentPlans extends Widget<PaymentPlanSettings> {
   private results: IEligibility[]
   private loading: boolean
   private fetchError: boolean
-
+​
```
Comment (syjust) [inline src/widgets/PaymentPlans/PaymentPlans.tsx:14]: remove line please (modify only part of code you have to)

File: src/widgets/PaymentPlans/PaymentPlans.tsx [hunk 8]
```diff
@@ -36,24 +36,26 @@ export class PaymentPlans extends Widget<PaymentPlanSettings> {
       },
     }
   }
-
+​
```
Comment (syjust) [inline src/widgets/PaymentPlans/PaymentPlans.tsx:39]: remove line please (modify only part of code you have to)

File: src/widgets/PaymentPlans/PaymentPlans.tsx [hunk 9]
```diff
@@ -36,24 +36,26 @@ export class PaymentPlans extends Widget<PaymentPlanSettings> {
...
   constructor(almaClient: Client, settings: SettingsLiteral<PaymentPlanSettings>) {
     super(almaClient, settings)
     this.results = []
     this.loading = false
     this.fetchError = false
   }
-
+​
```
Comment (syjust) [inline src/widgets/PaymentPlans/PaymentPlans.tsx:46]: remove line please (modify only part of code you have to)

File: src/widgets/PaymentPlans/PaymentPlans.tsx [hunk 10]
```diff
@@ -36,24 +36,26 @@ export class PaymentPlans extends Widget<PaymentPlanSettings> {
...
   private async fetchResults() {
     // Start with an empty list of results
     const results: Array<Eligibility | number> = []
-
+​
     const { purchaseAmount, plans } = this.config
     const installmentsCountsToQuery = []
-
+​
```
Comment (syjust) [inline src/widgets/PaymentPlans/PaymentPlans.tsx:53]: remove line please (modify only part of code you have to)

File: src/widgets/PaymentPlans/PaymentPlans.tsx [hunk 11]
```diff
@@ -89,14 +98,14 @@ export class PaymentPlans extends Widget<PaymentPlanSettings> {
       this.fetchError = true
       eligibilities = []
     }
-
+​
```
Comment (syjust) [inline src/widgets/PaymentPlans/PaymentPlans.tsx:101]: remove line please (modify only part of code you have to)

File: src/widgets/PaymentPlans/PaymentPlans.tsx [hunk 12]
```diff
@@ -89,14 +98,14 @@ export class PaymentPlans extends Widget<PaymentPlanSettings> {
...
     this.results = results
       .map((r) => (typeof r === 'number' ? eligibilities[r] : r))
       // In case of network error, eligibilities might be empty and results contain `undefined`
       // entries as a consequence, so make sure to filter them out
       .filter(Boolean)
   }
-
+​
```
Comment (syjust) [inline src/widgets/PaymentPlans/PaymentPlans.tsx:108]: remove line please (modify only part of code you have to)

File: src/widgets/PaymentPlans/PaymentPlans.tsx [hunk 13]
```diff
@@ -106,12 +115,12 @@ export class PaymentPlans extends Widget<PaymentPlanSettings> {
         this.render()
       })
     }
-
+​
```
Comment (syjust) [inline src/widgets/PaymentPlans/PaymentPlans.tsx:118]: remove line please (modify only part of code you have to)

File: src/widgets/PaymentPlans/PaymentPlans.tsx [hunk 14]
```diff
@@ -106,12 +115,12 @@ export class PaymentPlans extends Widget<PaymentPlanSettings> {
...
     const retry = () => {
       this.fetchError = false
       this.render()
     }
-
+​
```
Comment (syjust) [inline src/widgets/PaymentPlans/PaymentPlans.tsx:123]: remove line please (modify only part of code you have to)

**Review by syjust — CHANGES_REQUESTED** [review]

**Comment by xenosnikos** [comment]
@syjust  i think i covered all the formatting issues, i will be careful with this in future. 
let me know if i broke something :S

**Pull request closed without merging**